TKeazirian.HTTPServer/Response/Response.cs

```diff
@@ -1,13 +1,14 @@
 namespace TKeazirian.HTTPServer.Response;
 
+using Helpers;
+
 public class Response
 {
     public readonly string ResponseStatusLine;
     public readonly string ResponseHeaders;
-    public readonly string ResponseBody;
+    public readonly byte[]? ResponseBody;
```

Comment:
?

Reply:
It has to have the `?` with it here because it is possibly null.  C# thing I guess...at least that's what my editor is telling me.

Reply:
if I remember correctly, you had the same issue with `string?` at some point, and we spent some time and it turns out you did not need an option type.  I think it would be worth it to do the same thing here.

Reply:
I did!  I think I need it here - I've taken it away from several places, but I have logic to determine if it's null or not, so in this case, it may be null (if the body does not get set) and it may not be and that's what the `?` is for, I believe.